certified-connectors/Gratavid/apiDefinition.swagger.json

```diff
@@ -0,0 +1,542 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Gratavid",
+    "description": "Gratavid is a SaaS service that allows organizations to collect and send personalized videos.",
+    "version": "1.0",
+    "contact": {
+      "name": "Gratavid Support",
+      "url": "https://gratavid.zendesk.com/",
+      "email": "contact@gratavid.com"
+    }
+  },
+  "host": "api.gratavid.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/api/manageIntegrations": {
+      "x-ms-notification-content": {
+        "description": "Id Response",
+        "schema": {
+          "type": "object",
+          "properties": {
+            "noteTitle": {
+              "type": "string",
+              "description": "noteTitle",
+              "title": "Gratavid Note Title"
+            },
+            "noteURL": {
+              "type": "string",
+              "description": "noteURL",
+              "title": "Gratavid Note URL"
+            },
+            "contact_customUserId": {
+              "type": "string",
+              "description": "contact_customUserId",
```

Comment:
@Wtrapp All the description values of properties should be free from grammatical and spelling mistakes.

Ex.: `"description": "Custom UserId of the contact"`